applications/FluidDynamicsApplication/python_scripts/apply_two_fluids_inlet_process.py

```diff
@@ -0,0 +1,219 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+
+def Factory(settings, Model):
+    if( not isinstance(settings, KratosMultiphysics.Parameters) ):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyTwoFluidsInletProcess(Model, settings["Parameters"])
+
+
+class ApplyTwoFluidsInletProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        # settings for inlet with interface between fluids and separate velocities
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "mesh_id" : 0,
+            "model_part_name" : "user_provided_inlet_model_part_name",
```

Comment:
I'd use `"model_part_name": ""`as default. In this way you can check if the user-provided name is empty.

Reply:
Emptied.

Reply:
Did you implement the check as well?